homeassistant/components/dreo/config_flow.py

```diff
@@ -0,0 +1,55 @@
+"""Config flow to configure Dreo."""
+import logging
+import voluptuous as vol
+from hscloud.hscloud import HsCloud
+from collections import OrderedDict
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+from .const import DOMAIN
+import hashlib
+
+_LOGGER = logging.getLogger(__name__)
+_LOGGER.setLevel(logging.INFO)
+
+
+class DreoFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a Dreo config flow."""
+
+    def __init__(self) -> None:
+        """Instantiate config flow."""
+        self._username = None
+        self._password = None
+        self.data_schema = OrderedDict()
```

Comment:
- Since Python 3.7 dicts are ordered, the use of OrderedDict is no longer needed.
- This schema is static, lets put the schema in a constant instead